presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -361,9 +361,11 @@ public static HiveColumnStatistics fromMetastoreApiColumnStatistics(ColumnStatis
         }
         if (columnStatistics.getStatsData().isSetBooleanStats()) {
             BooleanColumnStatsData booleanStatsData = columnStatistics.getStatsData().getBooleanStats();
+            OptionalLong numFalse = fromMetastoreNumFalse(booleanStatsData.getNumFalses());
```

Comment:
can you be consistent and either use a method for both numTrue and numFalse or a ternary operator for both?  The logic between the two is very similar, but that's hard to see when they are implemented differently.  Personally, I prefer the ternary operator for both (just be sure to move the comment above the method up to here), but either way is fine with me.

Reply:
@rschlussel  Thank you very much for your detailed review;
For the illegal value generated by Impala, `numTrue` is `1` while `numFalse` is `-1`. So, we I get a `1` for `numTrue`, I cannot tell whether it is a legal value or an illegal value generated by Impala. Since from the Impala logic, the `numFalse=-1` and `numTrue =1` is set at the same time, so I have to tell whether the `numTrue=1` is illegal or not by checking whether `numFlase == -1`
So, I didn't quite understanding what  the ternary operation means;
Could you give me more information and suggestions?